transport-classes-io_uring/src/main/java/io/netty/incubator/channel/uring/IOUringEventLoop.java

```diff
@@ -154,8 +154,11 @@ protected void run() {
         final IOUringCompletionQueue completionQueue = ringBuffer.ioUringCompletionQueue();
         final IOUringSubmissionQueue submissionQueue = ringBuffer.ioUringSubmissionQueue();
 
-        // Lets add the eventfd related events before starting to do any real work.
+        // Let's add the eventfd related events before starting to do any real work.
         addEventFdRead(submissionQueue);
+        // We also need to submit this work because for short-lived event loops its possible
+        // to never enter a submitAndWait() call before shutting dwn.
+        submissionQueue.submit();
```

Comment:
It's not obvious to me that this guarantees a flush of the eventFd read. Opinions appreciated.

Reply:
The `submit()` method returns the number of commands that were flushed. There should not be any other commands submitted before this, but in case there is we can assert that the `submissionQueue.count()` is zero.